prov/rxd/src/rxd.h

```diff
@@ -192,6 +192,7 @@ struct rxd_ep {
 	struct dlist_entry rx_list;
 	struct dlist_entry rx_tag_list;
 	struct dlist_entry active_peers;
+	struct dlist_entry waiting_peers;
```

Comment:
If you call this rts_sent_list, it would be clearer what the state of the entries are that are in the list and what they are waiting on.